yb-voyager/cmd/exportDataDebezium.go

```diff
@@ -403,10 +410,29 @@ func reportStreamingProgress() {
 	}
 }
 
+func updateCallhomeEventsStats() {
+	//In case the pb is disabled, this will update the stats in 15 minutes
+	for {
```

Comment:
How about we structure the flow like this :
```
if disablepb = false or sendDiagnostics = true:
    calculateStreamingProgress() // this will calculate and update vars
if disablepb = false:
    reportStreamingPRogress() // this will actually print the table.
```